packages/wasm-terminal/lib/process/process.ts

```diff
@@ -1,11 +1,13 @@
-import WASI from "@wasmer/wasi";
+import { WasmFs } from "@wasmer/wasmfs";
```

Comment:
Same major API change here 😢

Reply:
Yeah, I think it's necessary :( (since we want to merge our changes eventually into upstream)

Reply:
This is not neccessary, we can make a PR to memfs that does not make this change, and allows us to contribute upstream 😄